hedera-node/src/test/java/com/hedera/services/store/models/UniqueTokenTest.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * -
+ * ‌
+ * Hedera Services Node
+ * Copyright (C) 2018 - 2021 Hedera Hashgraph, LLC
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.services.store.models;
+
+import com.hedera.services.state.submerkle.RichInstant;
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+class UniqueTokenTest {
+
+	@Test
+	void testSubject(){
```

Comment:
Maybe we can rename the test to `objectContractWorks` or have 2 tests: `settersWork` and `gettersWorks`, because at the moment we have 1 general `testSubject` that is kind of ambiguous what it actually tests.